mithril-explorer/src/components/PendingCertificate/index.js

```diff
@@ -68,10 +71,11 @@ export default function PendingCertificate(props) {
                 ? <div>No Signers registered</div>
                 : <>
                   <ListGroup variant="flush">
-                    <ListGroup.Item><b>Party id</b></ListGroup.Item>
+                    <ListGroup.Item><b>Pools</b></ListGroup.Item>
                     {pendingCertificate.signers.map(signer =>
                       <ListGroup.Item key={signer.party_id}>
-                        {signer.party_id}
+                        <PoolTicker partyId={signer.party_id} aggregator={aggregator}/><br/>
```

Comment:
Is this displaying correctly when we don't know the pool ticker?

Reply:
Actually, there is little chance that a pool ticker is not showing up after `1` epoch, so not a real problem after all :+1:

Reply:
We changed it so it will show `Not available` in that case (keeping the link to CExplorer):

![image](https://github.com/input-output-hk/mithril/assets/790521/e9fec7f5-7acc-4a4d-a23f-4591a98009d1)

There's still one case that will show nothing but a blank line: if on a network that is not mainnet, preprod or preview and the `/signers/tickers` route yield nothing (that should be the case since we have no data to import).